### PR TITLE
Added two free JavaScript courses in Hindi

### PR DESCRIPTION
**Pull request by Anuj-Gill**
### Description
Added 2 new courses in the JavaScript section of free-courses-hi.md, namely **Chai aur JavaScript** and **JavaScript Tutorial for beginners in Hindi / Urdu** on YouTube.

### Why is this valuable (or not)?

The advantages of these two courses are :

1. They are well structured
2. Both the courses go at a decent pace. They start with very basics and go on to some advanced concepts.
3. They also include some projects for better understanding of the concepts  and also gives hands-on experience.
4. Sets a base to learn frameworks/libraries like ReactJS etc. in future.

### How do we know it's really free?
Both of these courses are freely available on YouTube.

### For book lists, is it a book? For course lists, is it a course? etc.
They are courses.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -318,10 +318,12 @@
 
 ### JavaScript
 
+* [Chai aur Javascript | हिन्दी](https://www.youtube.com/playlist?list=PLu71SKxNbfoBuX3f4EOACle2y-tRC5Q37) - Chai aur Code
```
Comment (eshellman) [inline courses/free-courses-hi.md:321]:
```suggestion
* [Chai aur Javascript \| हिन्दी](https://www.youtube.com/playlist?list=PLu71SKxNbfoBuX3f4EOACle2y-tRC5Q37) - Chai aur Code
```

Comment (eshellman) [reply courses/free-courses-hi.md:321]: when you've made changes, please add `(changes made)` to the title of your PR

Comment (Anuj-Gill) [reply courses/free-courses-hi.md:321]: Ok, will keep this in mind form now onwards.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**